Product Designer/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+       <title>
+            Product Designer
+       </title>
+       <link rel="stylesheet" href="/Product Designer/assets/css/styles.css" />
```

Comment:
Will the Responsive mode work with viewport meta tag ? Did you really test them ?

Reply:
Yes Its working and I added the meta tag now ...